pkg/bgpv1/gobgp/conversions_test.go

```diff
@@ -0,0 +1,44 @@
+package gobgp
+
+import (
+	"context"
+	"testing"
+
+	gobgp "github.com/osrg/gobgp/v3/api"
+	"github.com/osrg/gobgp/v3/pkg/server"
+	"github.com/stretchr/testify/require"
+)
+
+// Test common conversions appear in the codebase
+func TestToGoBGPPath(t *testing.T) {
+	for _, tt := range commonPaths {
+		t.Run(tt.name, func(t *testing.T) {
+			s := server.NewBgpServer()
+			go s.Serve()
+
+			err := s.StartBgp(context.TODO(), &gobgp.StartBgpRequest{
+				Global: &gobgp.Global{
+					Asn:        65000,
+					RouterId:   "127.0.0.1",
+					ListenPort: -1,
+				},
+			})
+			require.NoError(t, err)
+
+			t.Cleanup(func() {
+				s.Stop()
+			})
+
+			path, err := ToGoBGPPath(&tt.path)
+			require.NoError(t, err)
+
+			res, err := s.AddPath(context.TODO(), &gobgp.AddPathRequest{
+				Path: path,
+			})
+			require.NoError(t, err)
+			require.NotZero(t, res.Uuid)
+		})
+	}
+}
+
+// TODO: Test ToAgentPath
```

Comment:
Could you address this TODO?

Reply:
I believe it is addressed, I extended the existing test to perform the conversion in both directions and compare the result with the original Path - see https://github.com/cilium/cilium/blob/8736d6daacf19a1692bfde109d135c4bfac19fca/pkg/bgpv1/gobgp/conversions_test.go#L48

Reply:
Oh, sorry. I have missed that.